pwnlib/abi.py

```diff
@@ -102,13 +124,22 @@ class SigreturnABI(SyscallABI):
     returns = False
 
 
-linux_i386   = ABI('esp', [], 4, 0)
-linux_amd64  = ABI('rsp', ['rdi','rsi','rdx','rcx','r8','r9'], 8, 0)
-linux_arm    = ABI('sp', ['r0', 'r1', 'r2', 'r3'], 8, 0)
-linux_aarch64 = ABI('sp', ['x0', 'x1', 'x2', 'x3'], 16, 0)
-linux_mips  = ABI('$sp', ['$a0','$a1','$a2','$a3'], 4, 0)
-linux_ppc = ABI('sp', ['r3', 'r4', 'r5', 'r6', 'r7', 'r8', 'r9', 'r10'], 4, 0)
-linux_ppc64 = ABI('sp', ['r3', 'r4', 'r5', 'r6', 'r7', 'r8', 'r9', 'r10'], 8, 0)
+# System V ABI used by many UNIXes
```

Comment:
Nit: This diff ends up being harder to read / check because of the rename.  Please leave the originals as `linux_xxx` and set `sysv_xxx = linux_xxx`.  This is less "correct" but I'd prefer it still

Reply:
Done